magstock/__init__.py

```diff
@@ -57,7 +65,7 @@ def food_cost(self):
 
     @property
     def auto_food(self):
-        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE] or hasattr(self, "auto_food_extra") and self.auto_food_extra
```

Comment:
Yeah, er, why shouldn't the magstock plugin know anything about bands? Event plugins are the "last in line" and can have as many dependencies as we want.

There SHOULD be a one-way interface - magstock should know stuff about bands, bands shouldn't know anything about magstock. Unless we expect to use auto_food_extra in multiple plugins (which right now we don't), you're adding a property to the band plugin that won't be used unless we have the magstock plugin running alongside it - it doesn't make that much sense to do that. Like, imagine someone installs the band plugin in their event. They see the `auto_food_extra` property and they go looking around for where it's used in the band plugin... except, it isn't used, leaving the poor coder confused.

If it wasn't so dependent on event-specific business logic - or if the concept of 'free food' was integral to bands - it'd be a different story.